### PR TITLE
Test: Replace decrecated $eval usages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Test: Replace decrecated $eval usages
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | No tests
| UI Tests          | https://github.com/boubkerbribri/ga.tests.ui.pr/actions/runs/7167634430
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/FO/FObasePage.ts [hunk 1]
```diff
@@ -650,10 +650,11 @@ export default class FOBasePage extends CommonPage {
...
   async getFooterLinksTextContent(page: Page, position: number): Promise<Array<string>> {
-    return page.$$eval(
-      this.wrapperSubmenuItemLink(position),
-      (all) => all.map((el) => (el.textContent ?? '').trim()),
-    );
+    return page
+      .locator(this.wrapperSubmenuItemLink(position))
+      .evaluateAll(
+        (all) => all.map((el) => (el.textContent ?? '').trim()),
+      );
```
Comment (Progi1984) [inline tests/UI/pages/FO/FObasePage.ts]:
```suggestion
      .locator(this.wrapperSubmenuItemLink(position))
      .allTextContents()
      .map((el) => el.trim());
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Can you revert changes in package-lock.json ?

File: tests/UI/pages/FO/FObasePage.ts (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/FO/FObasePage.ts]: done

**Comment by boubkerbribri** [comment]
> Can you revert changes in package-lock.json ?

done

**Review by Progi1984 — DISMISSED** [review]

**Review by nesrineabdmouleh — DISMISSED** [review]

File: tests/UI/pages/FO/FObasePage.ts (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/FO/FObasePage.ts]: I didn't see your last version so i did not add the trim, doesn't work. so fixed now

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by nesrineabdmouleh** [comment]
Thank you @boubkerbribri

**Pull request merged**